packages/components/dialog/src/dialog.ts

```diff
@@ -69,6 +110,21 @@ export class Dialog extends ScopedElementsMixin(LitElement) {
 
     // Disable scrolling while the dialog is open
     document.documentElement.style.overflow = 'hidden';
+
+    this.dialog?.setAttribute('closing', 'false');
+
+    /** Workaround for the backdrop background */
```

Comment:
Please provide a little more explanation why this is necessary for our future selves ;)